pennylane/operation.py

```diff
@@ -826,17 +811,6 @@ def __init__(self, *params, wires=None, do_queue=True, id=None):
         self._inverse = False
         super().__init__(*params, wires=wires, do_queue=do_queue, id=id)
 
-        # check the grad_method validity
-        if self.par_domain == "N":
-            assert (
-                self.grad_method is None
-            ), "An operation may only be differentiated with respect to real scalar parameters."
-        elif self.par_domain == "A":
-            assert self.grad_method in (
-                None,
-                "F",
-            ), "Operations that depend on arrays containing free variables may only be differentiated using the F method."
-
```

Comment:
This is deleted for good, since we cannot do this verification any more - we have to rely on the developers to do things right!